kratos/tests/cpp_tests/geometries/test_geometry.h

```diff
@@ -37,6 +37,7 @@ namespace Testing {
     typedef std::size_t IndexType;
 
     constexpr double EPSILON = std::numeric_limits<double>::epsilon();
+    constexpr double ZERO_TOLERANCE = 1e-16;
```

Comment:
Wouldn't this give us machine precision issues? @pooyan-dadvand @roigcarlo

Reply:
Do you mean 32bit vs 64 bit?¿

Reply:
I am not sure in 32bit but in 64bit it is ok